### ISSUE TITLE
Fetch all links in the line

### ISSUE THREAD

**Issue by Mikaela** [WildPHP/module-linksniffer#1]
```
2015-12-20 13:55:57+0200 <@Mikaela> maybe I should have linked http://blogs.msdn.com/b/powershell/archive/2015/06/03/looking-forward-microsoft-support-for-secure-shell-ssh.aspx & http://blogs.msdn.com/b/powershell/archive/2015/10/19/openssh-for-windows-update.aspx instead?
2015-12-20 13:55:59+0200 < FatalException> [http://is.gd/UdWNaG] OpenSSH for Windows Update - Windows PowerShell Blog - Site Home - MSDN Blogs | [second short url] and title are missing here
```

**Comment by NanoSector** [comment]
```
[12:57] <Mikaela> your bot ignored the second link, it should separate the short URLs and titles with | in the middle and Limnoria has the same issue
[12:58] <NanoSector> i deliberately made it like that
[12:58] <Mikaela> why? :(
[12:58] <Mikaela> where is the issue tracker?
[12:58] <NanoSector> http://github.com/WildPHP/module-linksniffer
[12:58] <FatalException> [http://is.gd/ru9XWy] WildPHP/module-linksniffer · GitHub
[12:59] <NanoSector> because imo there's no clean way to implement multiple links
[12:59] <NanoSector> you'd either have to send multiple replies to the channel or separate them and risk getting the message split in multiple lines
```

tl;dr: Not sure if there's a clean way to properly implement this.

**Comment by NanoSector** [comment]
With the backends system this should be pretty easy to implement using the multiple-message method that more bots implement. With the async method used right now this should pose no (significant) issues.
